ios/NativeSigner.xcodeproj/project.pbxproj

```diff
@@ -1661,7 +1634,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = io.parity.NativeSigner;
 				PRODUCT_NAME = NativeSigner;
 				SWIFT_OBJC_BRIDGING_HEADER = "NativeSigner/NativeSigner-Bridging-Header.h";
-				SWIFT_VERSION = 5.0;
+				SWIFT_VERSION = 4.2;
```

Comment:
Why downgrade swift version here?

Reply:
I was testing with an old Xcode that didn't support 5.  This change isn't important, I actually forgot about it.